auth/github.go

```diff
@@ -142,7 +150,7 @@ func (gum *GithubUserManager) GetLoginCallbackHandler() http.HandlerFunc {
 			grip.Errorf("Error sending code and authentication info to Github: %+v", err)
 			return
 		}
-		SetLoginToken(githubResponse.AccessToken, w)
+		SetLoginToken(githubResponse.AccessToken, gum.LoginDomain, w)
```

Comment:
why is this only necessary for the GitHub user manager?

Reply:
I'm just adding the ability to store a cookie with a configurable domain in all places that store the session cookie. I'm not actually sure this does anything, but nothing uses github auth anyway